File: code/js/src/views/user/CompaniesViews/models/Trainer.ts
class Trainer {
    id: string
    name: string
    gender: string
    assignedTrainees: number
    capacity: number


    constructor(id: string, name: string, gender: string, assignedTrainees: number,capacity: number, ) {
        this.id = id;
        this.name = name;
        this.gender = gender;
        this.assignedTrainees = assignedTrainees;
        this.capacity = capacity;
    }
}


export default Trainer;

File: code/jvm/src/main/kotlin/pt/isel/leic/ptgest/domain/company/model/Trainer.kt
package pt.isel.leic.ptgest.domain.company.model

import pt.isel.leic.ptgest.domain.common.Gender
import java.util.UUID

data class Trainer(
    val id: UUID,
    val name: String,
    val gender: Gender,
    val assignedTrainees: Int,
    val capacity: Int
)
